frontend/pages/home_page.js

```diff
@@ -1,34 +1,73 @@
 import React, { Component, PureComponent } from 'react';
 import { Redirect } from 'react-router';
 import appRoutes from 'src/constants/routes';
+import http from 'src/frontend/services/http';
 
 export default class HomePage extends Component {
   state = {
     loading: false,
     error: '',
     redirectTo: '',
+    products: '',
   }
 
   goToProductNewPage = () => {
     this.setState({ redirectTo: appRoutes.productNew });
   }
 
-  render() {
-    const { error, redirectTo } = this.state;
-    if (redirectTo) return <Redirect push to={redirectTo} />;
+  fetchProducts = () => {
+    http.get('/product').then(res => {
+        this.setState({
+          products: res.products
+        })
+    }).catch(e => {
+        console.log('error', e);
+    });
+  }
+
+  componentDidMount() {
+    this.fetchProducts();
+  }
 
-    return (
-      <div className='home-page'>
-        <header className='home-page__header'>
-          <div className='content'>
-            <button className='green-button' onClick={this.goToProductNewPage}>
-              <i className='fas fa-plus' /> New Product
-            </button>
+  renderProducts = () => {
+    const { products } = this.state;
+    if(!products) { return (<div> No products </div>); }
```

Comment:
no need for squiggly brackets on 1 liner if conditions, and i like to have a space after if conditions, and I only add parentheses around html if it's multi-line:

```
if (!products) return <div>No Products</div>;
```